grid/chat/channels/maps/discord.suma

```diff
@@ -0,0 +1,183 @@
+profile = "chat/channels@1.0"
+provider = "discord"
+
+"""
+GetChannels map
+"""
+map GetChannels {
+    return map error if (input.server === undefined) {
```

Comment:
nitpicking: part of the Comlink map uses 4 spaces / 2 spaces for indentation

Reply:
I already updated indentation. Hopefully it's better now.